megamek/src/megamek/server/GameManager.java

```diff
@@ -18629,7 +18644,8 @@ else if ((entity.heat >= 14) && !entity.isShutDown()) {
                 r.add(boom);
                 if (entity.getCrew().hasActiveTechOfficer()) {
                     rollValue += 2;
-                    String rollCalc = rollValue + " [" + diceRoll.getIntValue() + " + 2]";;
+                    String rollCalc = rollValue + " [" + diceRoll.getIntValue() + " + 2]";
+                    ;
```

Comment:
may as well get rid of the extra semi-colon

Reply:
What in the what?